### PR TITLE
added japanese message

### PR DESCRIPTION
**Pull request by softark**

### CODE REVIEW

File: filters/AuthFilter.php [hunk 1]
```diff
@@ -47,6 +47,6 @@ protected function preFilter($filterChain)
 		if ($user->checkAccess($itemName, $this->params))
 			return true;
 
-		throw new CHttpException(401, 'Access denied.');
+		throw new CHttpException(401, Yii::t('AuthModule.main', 'Access denied.'));
```
Comment (magefad) [inline filters/AuthFilter.php:50]: remove this yii::t due error when use (search in issues)

Comment (softark) [inline filters/AuthFilter.php:50]: Ah, thanks for the comment. I will check the related issue.

**Comment by cniska** [comment]
Please create PR without the Yii::t() change because it crashes when the filter is used outside of the module itself.

**Comment by softark** [comment]
OK, I will.

**Pull request closed without merging**